packages/fetchai/skills/generic_buyer/dialogues.py

```diff
@@ -161,7 +161,9 @@ def __init__(self, **kwargs) -> None:
         :return: None
         """
         Model.__init__(self, **kwargs)
-        BaseFipaDialogues.__init__(self, self.context.agent_address)
+        BaseFipaDialogues.__init__(
+            self, self.context.agent_address, dialogue_class=FipaDialogue
+        )
```

Comment:
Specifying the dialogue object to be manipulated by `Dialogues` as the subclassed `FipaDialogue` above.